Example/Backpack/NavigationData.swift

```diff
@@ -236,14 +237,20 @@ class NavigationData: NSObject {
                 Item(name: "With divider without padding", value: .story(dividedCardStoryboard(story: .dividedVerticalNoPadding)))
                 Item(name: "With divider and corner style large", value: .story(dividedCardStoryboard(story: .dividedHorizontalCornerStyleLarge)))
             }
+            Group(name: "Chips") {
+              Item(name: "Default", value: .story(ChipStory.default.presentableStory))
+              Item(name: "With icons", value: .story(ChipStory.withIcons.presentableStory))
+              Item(name: "With background colour", value: .story(ChipStory.withBackgroundColor.presentableStory))
+              Item(name: "Filled", value: .story(ChipStory.filled.presentableStory))
+              Item(name: "Filled with background colour", value: .story(ChipStory.filledWithBackgroundColor.presentableStory))
+            }
```

Comment:
I wasn't 100% on how to do it, but we could also make this part of the `Story` protocol so that this logic could remain in `ChipStory.swift`. Then here we could just do:

```swift
ChipStory.storyGroup
```

..or similar.